### DISCUSSION TITLE
RabbitMQ version 4

### DISCUSSION THREAD

**Discussion by mymasse** [rabbitmq/cluster-operator#1746] [Q&A] [answered]
We're looking into deploying Rabbit with the operator but were wondering if it's supports deploying the new Rabbit 4.x releases. I see the default version deployed by the operator is 3.13.7 as per the 2.10.0 release notes but it's not clear if version 4 would work.

**Comment by mkuratczyk** [comment, marked as answer]
Generally speaking - there is no tight coupling between the Operator versions and RabbitMQ versions. There are some limits, but generally speaking you can use an older or newer version of RabbitMQ than the default the Operator deploys.

The latest operator works mostly fine with RabbitMQ 4.0. We found and fixed one issue but it's relatively minor:
https://github.com/rabbitmq/cluster-operator/issues/1734

Once RabbitMQ 4.0.3 ships, we'll also release the Operator which defaults to it.
